File: Easy/Armstrong Numbers/armstrong-numbers.py
#User function Template for python3

class Solution:
    def armstrongNumber (self, n):
        # code here 
        t = n
        s = 0
        while t:
            r = t%10
            s+=r**3
            t = t//10
        if s == n:
            return "true"
        return "false"    


#{ 
 # Driver Code Starts
#Initial Template for Python 3
if __name__ == '__main__':
    t = int(input())
    for _ in range(t):
        n = input()
        n = int(n)
        ob = Solution()
        print(ob.armstrongNumber(n))

# } Driver Code Ends

File: Easy/Armstrong Numbers/armstrong-numbers.cpp
//{ Driver Code Starts
// Initial Template for C++
#include <bits/stdc++.h>
using namespace std;

// } Driver Code Ends
// User function Template for C++
class Solution {
  public:
    string armstrongNumber(int n){
        // code here
        int temp = n;
        int s = 0;
        int r;
        while(temp){
            r = temp%10;
            s+=r*r*r;
            temp = temp/10;
        }
      if (s == n){
          return "Yes";
          
      }
      return "No";
    }
};

//{ Driver Code Starts.
int main() {
    int t;
    cin >> t;
    while (t--) {
        int n;
        cin >> n;
        Solution ob;
        cout << ob.armstrongNumber(n) << endl;
    }
    return 0;
}

// } Driver Code Ends